feature-configs/typical-baremetal/performance/performance_profile.patch.yaml

```diff
@@ -18,5 +18,5 @@ spec:
   hugepages:
     pages:
     - size: "1G"
-      count: 4
```

Comment:
for this one we need to change it to `10` and remove the node so we allocate 5 on every numa